scss/dialog/_theme.scss

```diff
@@ -5,15 +5,10 @@
 
 
     // Titlebar
-    .k-dialog-titlebar {
```

Comment:
The dialog is different from the window by design. However, we could introduce dialog variables in this file.